pkg/migrations/seq_num.go

```diff
@@ -1,30 +1,23 @@
 package migrations
 
 import (
-	"github.com/pkg/errors"
 	"github.com/stackrox/rox/pkg/features"
 	"github.com/stackrox/rox/pkg/migrations/internal"
-	"github.com/stackrox/rox/pkg/utils"
 )
 
 // CurrentDBVersionSeqNum is the current DB version number.
 // This must be incremented every time we write a migration.
 // It is a shared constant between central and the migrator binary.
 func CurrentDBVersionSeqNum() int {
-	// If PostgresDatastore is enabled, we consider all migrations to Postgres are following
-	// the current migrations.
-	// if features.PostgresDatastore.Enabled() {
-	//	return internal.CurrentDBVersionSeqNum + internal.PostgresDBVersionPlus
-	// }
+	if features.PostgresDatastore.Enabled() {
+		return internal.CurrentDBVersionSeqNum + internal.PostgresDBVersionPlus
+	}
 	return internal.CurrentDBVersionSeqNum
 }
 
 // CurrentDBVersionSeqNumWithoutPostgres is the current DB version number
 // without Postgres migrations. This function should only be used in testing
 // environment.
 func CurrentDBVersionSeqNumWithoutPostgres() int {
-	if !features.PostgresDatastore.Enabled() {
-		utils.Must(errors.New("Unexpected call, ROX_POSTGRES_DATASTORE is not true"))
-	}
-	return internal.CurrentDBVersionSeqNum
+	return internal.CurrentDBVersionSeqNum - 1
```

Comment:
Why the -1?

Reply:
Because all valid migration numbers are "< CurrentDBVersionSeqNum" and postgres n is the count.

Reply:
I get it.  It's fine.  I was thinking about it from a readability and maintenance perspective but I think the only way to get rid of it would be to start the first postgres migration at n_00_to_n01.  That is probably overkill and over thinking it.

Reply:
Can you add this as a comment to the function?